src/CMakeLists.txt

```diff
@@ -4,10 +4,12 @@ SET(NNS_EDGE_SRCS
     ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-data.c
     ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-event.c
     ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-internal.c
-    ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-log.c
     ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-util.c
     ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-queue.c
 )
+IF (NOT ENABLE_TIZEN)
+    SET(NNS_EDGE_SRCS ${NNS_EDGE_SRCS} ${NNS_EDGE_SRC_DIR}/nnstreamer-edge-log.c)
```

Comment:
Why is the ```nnstreamer-edge-log.c``` excluded when ```ENABLE_TIZEN``` is off?

Reply:
If not using Tizen or Android, nns_edge_print_log() in nnstreamer-edge-log.c is used.